src/crawl4ai_mcp.py

```
@@ -195,13 +199,47 @@ async def crawl4ai_lifespan(server: FastMCP) -> AsyncIterator[Crawl4AIContext]:
+                # Initialize support documentation service
+                support_doc_service = SupportDocumentationService(
+                    supabase_client=supabase_client, embeddings_service=embeddings_service
+                )
+                logger.info("✓ Support documentation service initialized")
+
+            except Exception as e:
+                logger.error(f"Failed to initialize support documentation services: {e}")
+                embeddings_service = None
+                support_doc_service = None
+        else:
+            logger.warning("OPENAI_API_KEY not configured - support documentation tools will be unavailable")
```

Pylint (3.3.7)

[convention] Line too long (109/100)

(C0301)